db/core/src/test/java/org/ethereum/beacon/db/DatabaseTest.java

```diff
@@ -0,0 +1,95 @@
+package org.ethereum.beacon.db;
+
+import org.ethereum.beacon.db.source.DataSource;
+import org.ethereum.beacon.db.source.impl.HashMapDataSource;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import tech.pegasys.artemis.util.bytes.BytesValue;
+
+import java.nio.file.InvalidPathException;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class DatabaseTest {
+
+  @Tag("FIX")
+  @ParameterizedTest
+  @MethodSource("invalidArgumentsProvider")
+  void testInvalidCreation(String path, long bufferLimitsInBytes) {
+    assertThatThrownBy(() -> Database.rocksDB(path, bufferLimitsInBytes))
+            .isInstanceOf(InvalidPathException.class);
+  }
+
+  @Test
+  void testInvalidCreationWithNull() {
+    assertThatThrownBy(() -> Database.rocksDB(null, 0)).isInstanceOf(NullPointerException.class);
+  }
+
+  private static Stream<Arguments> invalidArgumentsProvider() {
+    return Stream.of(Arguments.of("null", 0), Arguments.of("", 0), Arguments.of("0123", 0));
+  }
+
+  @ParameterizedTest
+  @MethodSource("validArgumentsProvider")
+  void testValidCreation(String path, long bufferLimitsInBytes) {
+    final Database database = Database.rocksDB(path, bufferLimitsInBytes);
+    assertThat(database).isNotNull();
+  }
+
+  private static Stream<Arguments> validArgumentsProvider() {
+    return Stream.of(
+            Arguments.of("/tmp", 0),
+            Arguments.of("/tmp", Long.MAX_VALUE),
+            Arguments.of("/tmp", Long.MIN_VALUE));
+  }
```

Comment:
This likely won't work on windows.
Additionaly, RocksDB creates files. It's a good idea to clean up, i.e. remove the directories, created by RocksDB.